### PR TITLE
Purge cluster/kubectl.sh from nearly all docs.

### PR DESCRIPTION
**Pull request by brendandburns**
Mark cluster/kubectl.sh as deprecated.

@kprobst @RichieEscarez @juliaferraioli

### CODE REVIEW

File: cluster/kubectl.sh [hunk 1]
```diff
@@ -18,6 +18,14 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+echo "-=-=-=-=-=-=-=-=-=-="
+echo "NOTE:"
+echo "kubectl.sh is deprecated and will be removed soon."
+echo "please replace all usage with calls to the kubectl"
+echo "binary and ensure that it is in your PATH."
```
Comment (kprobst) [inline cluster/kubectl.sh]: Looks like kubectl.sh, in addition to looking for the kubectl binary, also sets --context for GKE and --server for ubuntu. Do we want to warn about this as well or is this no longer necessary?

If it's still necessary and that particular wrapper is needed, maybe we can have a separate script or somehow bake this into the providers (or just warn).

File: examples/guestbook-go/README.md [hunk 2]
```diff
@@ -169,8 +169,9 @@ redis-slave    name=redis,role=slave    name=redis,role=slave    10.0.234.24
 To play with the service itself, find the external IP of the load balancer:
 
 ```shell
-$ cluster/kubectl.sh get services guestbook -o template --template='{{(index .status.loadBalancer.ingress 0).ip}}'
+$ kubectl get services guestbook -o template --template='{{(index .status.loadBalancer.ingress 0).ip}}'
 104.154.63.66$
+
```
Comment (kprobst) [inline examples/guestbook-go/README.md]: unnecessary empty line?

**Comment by bgrant0607** [comment]
cc @krousey

File: docs/getting-started-guides/libvirt-coreos.md [hunk 3]
```diff
@@ -116,7 +116,7 @@ virsh -c qemu:///system list
 You can check that the kubernetes cluster is working with:
 
 ```
-$ ./cluster/kubectl.sh get minions
+$ kubectl get minions
```
Comment (bgrant0607) [inline docs/getting-started-guides/libvirt-coreos.md]: kubectl get minions is gone

File: cluster/kubectl.sh (hunk 1 above)
Comment (brendandburns) [inline cluster/kubectl.sh]: I don't see the ```--context``` bits for GKE, for ubuntu, the ```--server``` flag is mostly deprecated in favor of kubectl contexts, so I think that it's ok to just drop these.  People should have the appropriate context files in ```.kube/config```

Comment (brendandburns) [inline cluster/kubectl.sh]: I also added to the deprecation note pointing people at ```kubectl help config```

File: examples/guestbook-go/README.md (hunk 2 above)
Comment (brendandburns) [inline examples/guestbook-go/README.md]: done.

File: docs/getting-started-guides/libvirt-coreos.md (hunk 3 above)
Comment (brendandburns) [inline docs/getting-started-guides/libvirt-coreos.md]: done.

**Comment by brendandburns** [comment]
Comments addressed, ptal.

Thanks!
--brendan

File: cluster/kubectl.sh (hunk 1 above)
Comment (kprobst) [inline cluster/kubectl.sh]: Looks good.

**Comment by kprobst** [comment]
LGTM

**Comment by krousey** [comment]
@brendandburns Please rebase

**Comment by brendandburns** [comment]
@krousey rebased.  ptal.

thanks!
--brendan

File: cluster/kubectl.sh (hunk 1 above)
Comment (jlowdermilk) [inline cluster/kubectl.sh]: fyi: #9227 removes the handling for GKE, ubuntu.

**Pull request merged**